### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/lit/commit/25378818c8ba91c8c588cccfa90e32583322afe6#commitcomment-87898999

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/lit/commit/9bbdf72f787441f2836faa5bf0710101e34ae95b

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-The Lit community is full of bright, thoughtful, helpful, supportive people - it's on all of us to keep it this way! Google and the Lit team want to help foster and support this, and we take any breaches of mutual respect very seriously. Below is the community code of conduct, which applies to our Github repos and organizations, mailing lists, Slack channels, and any other Lit-supported communication group.
+The Lit community is full of bright, thoughtful, helpful, supportive people - it's on all of us to keep it this way! Google and the Lit team want to help foster and support this, and we take any breaches of mutual respect very seriously. Below is the community code of conduct, which applies to our GitHub repos and organizations, mailing lists, Slack channels, and any other Lit-supported communication group.
```
Comment (jsoref) [inline CODE_OF_CONDUCT.md:1]: brand

File: README.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ lit-html 1.x source is available on the [`lit-html-1.x`](https://github.com/lit/
   - [`lit-starter-ts`](./packages/lit-starter-ts) ([template
     repo](https://github.com/lit/lit-element-starter-ts)) - A starter repo for building reusable components using Lit in TypeScript.
   - [`lit-starter-js`](./packages/lit-starter-js) ([template
-    repo](https://github.com/lit/lit-element-starter-js)) - A starter repo for building reusable components using Lit in Javascript.
+    repo](https://github.com/lit/lit-element-starter-js)) - A starter repo for building reusable components using Lit in JavaScript.
```
Comment (jsoref) [inline README.md:61]: brand

File: packages/labs/analyzer/CHANGELOG.md [hunk 3]
```diff
@@ -32,7 +32,7 @@
 
 - [#2804](https://github.com/lit/lit/pull/2804) [`e0517ccf`](https://github.com/lit/lit/commit/e0517ccf79d983a8d6ec53969f29e130830fe3e8) - Read property options from decorated properties
 
-- [#2896](https://github.com/lit/lit/pull/2896) [`3bd330f3`](https://github.com/lit/lit/commit/3bd330f3db4c2f618181b8602563db3ab879f33d) - Add utilties for getting LitElement declarations
```
Comment (jsoref) [inline packages/labs/analyzer/CHANGELOG.md:35]: Some projects don't like changing changelogs... happy to drop this or anything else.

File: packages/lit-html/CHANGELOG.md [hunk 4]
```diff
@@ -720,7 +720,7 @@ Changes below were based on the [Keep a Changelog](http://keepachangelog.com/) f
 
 - Refactored how template tags and `render()` are implemented so that all
   specialization of template syntax is done in tags, not `render()`, allowing
-  for the mixining of templates of different syntaxes, and for hooks in
+  for the mixing-in of templates of different syntaxes, and for hooks in
```
Comment (jsoref) [inline packages/lit-html/CHANGELOG.md:723]: notable

File: packages/lit-html/src/test/directive-helpers_test.ts [hunk 5]
```diff
@@ -106,7 +106,7 @@ suite('directive-helpers', () => {
     assertContent('<div>A</div>');
   });
 
-  test('insertPart keeps connectinon state in sync', () => {
+  test('insertPart keeps connection state in sync', () => {
```
Comment (jsoref) [inline packages/lit-html/src/test/directive-helpers_test.ts:109]: notable

**Review by jsoref — COMMENTED** [review]
Most corrections automatically suggested by Google Sheets. All fault mine.

**Review by AndrewJakubowicz — APPROVED** [review]
Thank you! I added a Changeset and added @arthurevans to double check that changes make sense.

**Pull request merged**